resotoclient/ca.py

```diff
@@ -54,46 +56,12 @@ def cert_to_bytes(cert: Certificate) -> bytes:
     return cert.public_bytes(serialization.Encoding.PEM)
 
 
-def write_ca_bundle(
-    cert: Certificate, cert_path: str, include_certifi: bool = True, rename: bool = True
-) -> None:
-    tmp_cert_path = f"{cert_path}.tmp" if rename else cert_path
-    with open(tmp_cert_path, "wb") as f:
-        if include_certifi:
-            f.write(certifi.contents().encode())
```

Comment:
This line is crucial for the client to work with non-Resoto certificates. Maybe I missed it, but I did not see a replacement for it.
In the current code the Resoto CA is just one of hundreds of CAs that the client supports.